docs/features/telemetry-enrichment.md

```diff
@@ -5,6 +5,14 @@ layout: default
 
 # Telemetry Enrichment
 
+## Installation
+
+This feature requires to install our NuGet package
+
+```shell
+PM > Install-Package Arcus.Observability.Telemetry.Serilog
```

Comment:
This makes me wonder if we shouldn't rename it to `Arcus.Observability.Telemetry.Serilog.Enrichers`

Reply:
Could be yeah. On the other hand; will we ever enrich Serilog sinks or other stuff to have another namespace for `...Telemetry.Serilog.XXX`?

Reply:
Yes, see #35 already introduces an emptry project

Reply:
Ok, then definitly moving to `...Enrichers`! I guess 🏭

Reply:
https://github.com/arcus-azure/arcus.observability/issues/36